server/api/users.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const router = express.Router();
+import { getAllUsers, getUserById }  from "../services/database/users.js";
+
+
+
+
+router.get("/", (_, res) => {
+	res.json({ message: "Hello, world!" });
+});
+
+router.get("/get-all", function (_, response) {
```

Comment:
if you have a users end point to get all users you would just use 

GET apiLink/users. 

It is not restfull to have the action name in the link